test/fixtures/shadow-dom-untabbable.html

```diff
@@ -0,0 +1,7 @@
+<test-shadow tabindex="-1">
+  <template shadowroot="open">
+    <div id="container">
+      <input id="input" type="text" />
+    </div>
+  </template>
+</test-shadow>
```

Comment:
I appreciate the new test, thank you! Could we also have another test that shows when the `nestedCandidates.length !== -1` case, that is, a test that checks the opposite of what this new test is checking?

Based on your explanation in the code:
```
  // If a custom element with a tabindex of -1 contains a single otherwise tabbable
  // element, browsers will not allow tab targeting said child.
  return nestedCandidates.length !== 1;
```
I'm getting the impression that if I changed your test to:
```
<test-shadow tabindex="-1">
  <template shadowroot="open">
    <div id="container">
      <input id="input" type="text" />
      <input id="input2" type="text" />
    </div>
  </template>
</test-shadow>
```
that now, both inputs would be tabbable? Seems a bit strange, so I wanted to double-check. Having a passing test that checks for the opposite is always a good reminder of all the corner cases.

Reply:
Definitely! Thanks for checking it out.